content/200-concepts/100-components/database-drivers.mdx

```diff
@@ -17,17 +17,28 @@ Prisma Client can connect and run queries against your database using JavaScript
 
 Prisma will use the Query Engine to transform the Prisma Client query to SQL and run the generated SQL queries via the JavaScript database driver.
 
-![Query flow from the user application to the database using Prisma Client and driver adapters](./images//drivers/qe-query-engine-adapter.png)
+![Query flow from the user application to the database using Prisma Client and driver adapters](./images/drivers/qe-query-engine-adapter.png)
 
-### Serverless driver adapters
+There are 2 different types of driver adapters:
 
-- [Neon](/guides/database/neon#how-to-use-the-neon-serverless-driver-with-prisma-preview)
-- [PlanetScale](/guides/database/planetscale#how-to-use-the-planetscale-serverless-driver-with-prisma-preview)
+- [Database driver adapters](#database-driver-adapters)
+- [Serverless driver adapters](#serverless-driver-adapters)
 
 ### Database driver adapters
 
+You can connect to your database using a Node.js-based driver from Prisma Client using a database driver adapter. Prisma maintains the following database driver adapters:
+
 - [Turso](/guides/database/turso#connect-and-query-your-primary-database)
 
+### Serverless driver adapters
+
+Database providers, such as Neon and PlanetScale, allow you to connect to your database using other protocols, such as HTTP and WebSockets. These database drivers are optimized for connecting to your database in serverless and edge environments.
```

Comment:
"other protocols" - other than TCP, other than database driver adapters do? Might be worth spelling out what the baseline is that we're referring to?